File: easy/2006. Count Number of Pairs With Absolute Difference K/2006. Count Number of Pairs With Absolute Difference K.go
package main

func absDiffInt(x, y int) int {
	if x < y {
		return y - x
	}
	return x - y
}

func countKDifference(nums []int, k int) int {
	res := 0
	n := len(nums)

	for i := 0; i < n; i++ {
		for j := i + 1; j < n; j++ {
			if absDiffInt(nums[i], nums[j]) == k {
				res += 1
			}
		}
	}

	return res
}

File: easy/2006. Count Number of Pairs With Absolute Difference K/2006. Count Number of Pairs With Absolute Difference K.py
from typing import List


class Solution:
    def countKDifference(self, nums: List[int], k: int) -> int:
        res, n = 0, len(nums)

        for i in range(n):
            for j in range(i + 1, n):
                if abs(nums[j] - nums[i]) == k:
                    res += 1

        return res
